### ISSUE TITLE
Support profiles requiring MFA

### ISSUE THREAD

**Issue by Glen-Moonpig** [mercury2269/sqsmover#24]
I am trying to use a profile for a role, based on a user role which requires an MFA token input.

I am using the `--profile` argument with sqsmover and getting this error:

```
$ ./sqsmover --source=my-source-queue --destination=my-destination-queue --profile="a-profile" --region="eu-west-1"

   ⨯ Unable to create AWS session for region
%!(EXTRA string=eu-west-1)
```

I assume the error results from the absense of MFA token as I have not been prompted to supply one and can't see an argument for it.

Credentials file

```
[default]
aws_access_key_id=JADKJMYKEY981234268
aws_secret_access_key=7823kvssecretdwk3jeoqw3ehfwiosjd
[a-role]
mfa_serial=arn:aws:iam::8276358562934:mfa/me@email.com
role_arn=arn:aws:iam::32412847024:role/a-role
source_profile=default
```

**Comment by Glen-Moonpig** [comment]
I see there is a related pull request #21. This may solve it, but not sure I understand the usage of that solution.

**Comment by mercury2269** [comment]
Should be resolved with 0.4.0 release.
